packages/components/src/public/QueryModel/GridPanel.tsx

```diff
@@ -1078,6 +1088,7 @@ export class GridPanel<T = {}> extends PureComponent<Props<T>, State> {
                         model={model}
                         onCancel={this.toggleCustomizeView}
                         onUpdate={this.onSessionViewUpdate}
+                        selectedColumn={this.state.selectedColumn}
```

Comment:
minor: add `selectedColumn` to the top of the render where other state variables are listed and then update this to:
`selectedColumn={selectedColumn}`

Reply:
Done.